x-pack/plugins/lists/server/services/exception_lists/utils/import/sort_exception_lists_to_create_update.ts

```diff
@@ -102,6 +104,27 @@ export const sortExceptionListsToUpdateOrCreate = ({
           type: savedObjectType,
         },
       ];
+    } else if (existingLists[listId] != null && generateNewListId) {
+      const attributes: ExceptionListSoSchema = {
+        ...existingLists[listId],
+        comments: undefined,
+        created_at: dateNow,
+        created_by: user,
+        description,
+        entries: undefined,
+        immutable: false,
+        item_id: undefined,
+        list_type: 'list',
+        tie_breaker_id: uuid.v4(),
+        updated_by: user,
+      };
+      results.listsToCreate = [
+        ...results.listsToCreate,
+        {
+          attributes,
+          type: savedObjectType,
+        },
+      ];
```

Comment:
This logic allows us to import a list + items that already exists as a **new** list. We then have to generate new list item id's for all of the items in the list that are being imported.